sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/CosmosPagedIterableTest.java

```diff
@@ -113,6 +117,121 @@ public void readAllItemsBySubscribeWithCosmosPagedIterableHandler() throws Excep
         assertThat(handleCount.get() >= 1).isTrue();
     }
 
+    @Test(groups = { "simple" }, timeOut = TIMEOUT, enabled = false)
+    public void queryItemsWithCosmosPagedIterable() throws Exception {
+
+        CosmosQueryRequestOptions cosmosQueryRequestOptions = new CosmosQueryRequestOptions();
+        cosmosQueryRequestOptions.setMaxBufferedItemCount(10);
+        CosmosPagedIterable<ObjectNode> cosmosPagedIterable = cosmosContainer.queryItems("select * from c",
+                cosmosQueryRequestOptions, ObjectNode.class);
+
+        Iterable<FeedResponse<ObjectNode>> feedResponses = cosmosPagedIterable.iterableByPage(10);
+        //  Just creating iterator drains all the results!
+        Iterator<FeedResponse<ObjectNode>> iterator = feedResponses.iterator();
+        if (iterator.hasNext()) {
+            FeedResponse<ObjectNode> next = iterator.next();
+            logger.info("Next is : {}", next.getResults().size());
```

Comment:
I did not see any validation here, should we validate we are getting all the pages etc

Reply:
Yes it's for some local validation. Since there are no asserts, I have disabled it.